techmatters/flex-plugins
# Transcript Clean up Tasks: align shape with actual exported transcripts

## Description
This PR aligns the shape of the transcripts in the frontend as they are actually exported in our system, alongside the pertinent adjustments to make the transcript view to work as intended.

### Checklist
](https://tech-matters.atlassian.net/browse/CHI-1631)- [ ] New tests added
- [n/a] Feature flags added
- [n/a] Strings are localized
- [n/a] Tested for call contacts

### Verification steps
- `npm run dev`
- Save a new chat contact, with several interactions from both ends, wait about a minute to make sure the transcript is exported.
- Search for the contact and expand the transcript section.
- Confirm that the styles are as intended (couselor messages at the right and blue bubble, non counselor at the left and grey bubble, consecutive messages from the same sender are grouped).
